release-notes.md

```diff
@@ -139,6 +139,20 @@ None.
 
 [/Details]
 
+[Details=1.11.5]
```

Comment:
Should this go here or at the top? 1.9.5 is above 1.10 because it was released later.